VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html

```diff
@@ -9,13 +9,12 @@
         >
             <span class='far fa-ellipsis-h'></span>
             <div *ngIf='showContextMenu' class='dropdown-content'>
-                <div class="govuk-heading-s horizontal-spacer">Hearing controls</div>
+                <div class="govuk-heading-s">{{'private-consultation-room-controls.hearing-controls' | translate}}</div>
+                <hr id="hr">
```

Comment:
This should be done with css.

Reply:
Just to be clear as to why. hr has a semantic meaning and should not be used for styling. This can cause issues for screen readers and would not be considered accessible. Additionally they may appear visually different in different browsers.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr
"The <hr> tag defines a thematic break in an HTML page (e.g. a shift of topic)."